pkg/console/operator/sync_v400.go

```diff
@@ -78,29 +80,31 @@ func sync_v400(co *ConsoleOperator, consoleConfig *v1alpha1.Console) (*v1alpha1.
 	}
 	toUpdate = toUpdate || depChanged
 
-	// handy debugging block
-	logrus.Printf("sync_v400 complete. updates made:")
-	logrus.Printf("service changed: %v \n", svcChanged)
-	logrus.Printf("route is new: %v \n", rtChanged)
-	logrus.Printf("configMap changed: %v \n", cmChanged)
-	logrus.Printf("secret changed: %v \n", secChanged)
-	logrus.Printf("oauth changed: %v \n", oauthChanged)
-	logrus.Printf("deployment changed: %v \n", depChanged)
-
 	// if any of our resources have changed, we should update the consoleConfig.Status. otherwise, skip this step.
 	if toUpdate {
-		logrus.Infof("Sync_v400: To update Spec: %v", toUpdate)
+		logrus.Infof("sync_v400: to update spec: %v", toUpdate)
 		// TODO: set the status.
 		// setStatus(consoleConfig.Status, svc, rt, cm, dep, oa, sec)
 	}
 
+	defer func() {
+		logrus.Printf("sync loop 4.0.0 complete:")
+		logrus.Printf("service changed: %v", svcChanged)
+		logrus.Printf("route is new: %v", rtChanged)
```

Comment:
Should this be "route changed"?

Reply:
Yeah, thinking about that. Route has been unique as I haven't done an 'update', just a 'create' or 'get' -- we just wait for `host` to be applied. So its a bit intentional, but it may be better to just say "changed" anyway.  I'll update that.

Reply:
updated